assets/css/admin.scss

```diff
@@ -3929,8 +3929,6 @@ img.help_tip {
 			top: 20px;
 		}
 
-		woocommerce-help-tip
-
```

Comment:
Is this something coming from the merge? Seems unrelated to the rest of the PR?

Reply:
@peterfabian merge conflicts, still this line sounds like a bug anyway.

Reply:
@claudiosanches Can you please push an update? Don't want to add commits myself as a reviewer...

Reply:
In master that line is there but seems out of place https://github.com/woocommerce/woocommerce/blob/master/assets/css/admin.scss#L3932

Reply:
This is the commit that introduced the string https://github.com/woocommerce/woocommerce/commit/64c71cb7243e6bc52ad1c782ce01bd38fc55e673#diff-6b2f6833e79a750bf7f3a84223058d3eR3927 Which seems odd as that string should not be present. I think it would be fine to remove it from this PR, linting probably cleaned it up.

Reply:
Thanks for investigating, @kloon. Let's keep the change as it is, then.